rector.php

```diff
@@ -6,16 +6,13 @@
 use Rector\DeadCode\Rector\StaticCall\RemoveParentCallWithoutParentRector;
 use Rector\Privatization\Rector\ClassMethod\PrivatizeFinalClassMethodRector;
 use Rector\TypeDeclaration\Rector\Property\TypedPropertyFromAssignsRector;
-use Rector\ValueObject\PhpVersion;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->paths([__DIR__.'/']);
-    $rectorConfig->skip([__DIR__.'/vendor']);
+    $rectorConfig->paths([__DIR__.'/src']);
```

Comment:
the `tests` need in the `paths()`, that will make invalid result:

```diff
1) tests/FooTest.php:10

    ---------- begin diff ----------
@@ @@
 {
     private Foo $foo;

-    protected function setUp(): void
+    private function setUp(): void
     {
-        parent::setUp();
-
         $this->foo = new Foo(234);
```

The `.phar` need to be loaded directly, otherwise, it never known, as there is no standard part which part that loaded in `.phar`, it can't magically know which part that should be loaded inside `.phar` itself.